lib/controllers/facebook-login.js

```diff
@@ -36,7 +38,7 @@ module.exports = function (req, res) {
 
   if (!req.query.access_token) {
     logger.info('A user attempted to log in via Facebook OAuth without specifying an OAuth token.');
-    return res.status(400).json({ message: 'access_token parameter required.' });
+    return oauth.errorResponder(req, res, { message: 'access_token parameter required.' });
```

Comment:
To be consistent, maybe always pass an error object to `oauth.errorResponder()`? I.e.

```javascript
return oauth.errorResponder(req, res, new Error('access_token parameter required.'));
```